IndexFollowingIT.testCleanUpShardFollowTasksForDeletedIndices fails once a week

<!--
Please fill out the following information, and ensure you have attempted
to reproduce locally
-->

**Build scan**: https://gradle-enterprise.elastic.co/s/4efglek36fhde/tests/failed

**Repro line**:
```
./gradlew ':x-pack:plugin:ccr:internalClusterTest' \
  --tests "org.elasticsearch.xpack.ccr.IndexFollowingIT.testCleanUpShardFollowTasksForDeletedIndices" \
  -Dtests.seed=CE4A0D350ACA4A22 -Dtests.security.manager=true \
  -Dtests.locale=es-PA -Dtests.timezone=NZ -Druntime.java=11
```

**Reproduces locally?**: no

**Applicable branches**: 7.8-master

**Failure history**:
https://build-stats.elastic.co/goto/0d10972e7446c8b18341f71ff20e62da

**Failure excerpt**:
```
  1> [2020-10-29T07:27:12,533][WARN ][o.e.p.PersistentTasksClusterService] [follower2] The task [OeNv83z4TVaJbMYHYMgo2Q-0] wasn't found, status is not updated
  1> [2020-10-29T07:27:12,533][WARN ][o.e.x.c.a.ShardFollowTaskCleaner] [follower2] failed to clean up task [OeNv83z4TVaJbMYHYMgo2Q-0]
  1> org.elasticsearch.ResourceNotFoundException: the task with id [OeNv83z4TVaJbMYHYMgo2Q-0] and allocation id [1] not found
  1> 	at org.elasticsearch.persistent.PersistentTasksClusterService$2.execute(PersistentTasksClusterService.java:171) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.ClusterStateUpdateTask.execute(ClusterStateUpdateTask.java:59) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:697) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.MasterService.calculateTaskOutputs(MasterService.java:319) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.MasterService.runTasks(MasterService.java:214) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.MasterService$Batcher.run(MasterService.java:151) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.TaskBatcher.runIfNotProcessed(TaskBatcher.java:150) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.cluster.service.TaskBatcher$BatchedTask.run(TaskBatcher.java:188) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:674) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:252) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:215) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
  1> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
  1> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
  1> 	at java.lang.Thread.run(Thread.java:834) [?:?]
  1> [2020-10-29T07:27:13,883][INFO ][o.e.c.m.MetadataDeleteIndexService] [leader1] [index1/IyIChlKwR-yubWuGq-YHGQ] deleting index
  1> [2020-10-29T07:27:14,505][INFO ][o.e.x.c.IndexFollowingIT ] [testCleanUpShardFollowTasksForDeletedIndices] after test
  2> REPRODUCE WITH: ./gradlew ':x-pack:plugin:ccr:internalClusterTest' --tests "org.elasticsearch.xpack.ccr.IndexFollowingIT.testCleanUpShardFollowTasksForDeletedIndices" -Dtests.seed=CE4A0D350ACA4A22 -Dtests.security.manager=true -Dtests.locale=es-PA -Dtests.timezone=NZ -Druntime.java=11
  2> java.lang.AssertionError: 
    Expected: a collection with size <0>
         but: collection size was <1>
        at __randomizedtesting.SeedInfo.seed([CE4A0D350ACA4A22:65CBC8E816F78A7B]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.ccr.IndexFollowingIT.lambda$testCleanUpShardFollowTasksForDeletedIndices$60(IndexFollowingIT.java:1387)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:950)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:923)
        at org.elasticsearch.xpack.ccr.IndexFollowingIT.testCleanUpShardFollowTasksForDeletedIndices(IndexFollowingIT.java:1384)
  1> [2020-10-29T07:27:14,668][INFO ][o.e.x.c.IndexFollowingIT ] [testFollowIndexWithConcurrentMappingChanges] before test
  1> [2020-10-29T07:27:14,672][INFO ][o.e.t.InternalTestCluster] [testFollowIndexWithConcurrentMappingChanges] changing cluster size from 5 data nodes to 2
  1> [2020-10-29T07:27:14,674][INFO ][o.e.t.InternalTestCluster] [testFollowIndexWithConcurrentMappingChanges] adding voting config exclusions [follower0, follower1] prior to restart/shutdown
  1> [2020-10-29T07:27:17,754][INFO ][o.e.n.Node               ] [testFollowIndexWithConcurrentMappingChanges] stopping ...
  1> [2020-10-29T07:27:17,768][INFO ][o.e.c.c.Coordinator      ] [follower0] master node [{follower2}{wxc1uojJQL6bSc1yBYq3IQ}{qrkdDLhfQeSChddhzEet6Q}{127.0.0.1}{127.0.0.1:42919}{chimrsw}{xpack.installed=true}] failed, restarting discovery
  1> org.elasticsearch.transport.NodeDisconnectedException: [follower2][127.0.0.1:42919][disconnected] disconnected
  1> [2020-10-29T07:27:17,780][INFO ][o.e.c.s.MasterService    ] [follower2] node-left[{follower0}{QrWTOiNnRFC39lRDPxBDbQ}{lwG_aLNtQUa4JZHITB9YrQ}{127.0.0.1}{127.0.0.1:45799}{chimrsw}{xpack.installed=true} reason: disconnected], term: 1, version: 34, delta: removed {{follower0}{QrWTOiNnRFC39lRDPxBDbQ}{lwG_aLNtQUa4JZHITB9YrQ}{127.0.0.1}{127.0.0.1:45799}{chimrsw}{xpack.installed=true}}
  1> [2020-10-29T07:27:17,782][INFO ][o.e.n.Node               ] [testFollowIndexWithConcurrentMappingChanges] stopped
  1> [2020-10-29T07:27:17,782][INFO ][o.e.n.Node               ] [testFollowIndexWithConcurrentMappingChanges] closing ...
  1> [2020-10-29T07:27:17,840][INFO ][o.e.n.Node               ] [testFollowIndexWithConcurrentMappingChanges] closed
```

--- a/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
+++ b/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
@@ -1385,7 +1385,7 @@ public class IndexFollowingIT extends CcrIntegTestCase {
             String action = ShardFollowTask.NAME + "[c]";
             ListTasksResponse listTasksResponse = followerClient().admin().cluster().prepareListTasks().setActions(action).get();
             assertThat(listTasksResponse.getTasks(), hasSize(0));
-        });
+        }, 60, TimeUnit.SECONDS);
         ensureNoCcrTasks();
     }
 

